### ISSUE TITLE
inefficient polling in cvat ui (api/lambda/requests)

### ISSUE THREAD

**Issue by alx-** [opencv/cvat#2707]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

![2021-01-22 23_31_46-DevTools - cvat org_tasks_create](https://user-images.githubusercontent.com/1101458/105544222-5fe41400-5d03-11eb-99dd-65da77eb7148.png)

I registered a new user on cvat.org. Logged in and immediately noticed that UI was continuously making a whole lot of unusual API requests.

### Expected Behaviour
UI should not make extraneous API calls to check for anything unrelated to user's activity. 

If there is a global task queue, it doesn't make sense to poll for its results in each user's UI, does it?

### Current Behaviour
I was curious, so I did some digging. Turns out the culprit was this piece of code:
```typescript
            const requests = await core.lambda.requests();
            requests
                .map((request: any): object => ({
                    taskID: +request.function.task,
                    requestID: request.id,
                }))
                .forEach((inferenceMeta: InferenceMeta): void => {
                    listen(inferenceMeta, dispatchCallback);
                });
```
/lambda/requests end-point does seem to return non-empty list of requests, unrelated to user's previous activity (which was precisely none). 

![2021-01-23 00_06_50-Request List – Django REST framework](https://user-images.githubusercontent.com/1101458/105546941-a4bd7a00-5d06-11eb-8c13-0572ddbdf738.png)

I assume this is a global queue for "lambda" requests. Not sure why cvat-UI needs to listen/poll on each and every one of them, even though these are completely irrelevant to other users.

### Steps to Reproduce (for bugs)
1. Register a new user at cvat.org
2. Confirm e-mail and log-in
3. Check out network tab in Dev Tools

**Comment by nmanovic** [comment]
@alx- , I agree that it has to be fixed.

**Comment by bsekachev** [comment]
Fixed in #6116
